router/state/manager.go

```diff
@@ -210,7 +210,7 @@ func (sm *ManagerImpl) LoadRouterModel(filePath string) {
 	model, err := common.NewReceiverRouterDataModelFromFile(filePath, RouterDataModelListerBufferSize)
 
 	if err != nil {
-		pfxlog.Logger().WithError(err).Errorf("could not load router model from file [%s]", filePath)
+		pfxlog.Logger().WithError(err).Infof("could not load router model from file [%s]", filePath)
```

Comment:
WithError but Info? this feels strange to be an info?

Reply:
It's usually not an error because on initial startup the file won't exist. If the file doesn't exist, we'll just get the full state from the controller again, so it doesn't prevent anything from working. Given that, it feels like it doesn't reach the level of error.